kimchi/src/prover.rs

```diff
@@ -179,10 +183,11 @@ where
             // padding
             w.extend(std::iter::repeat(ScalarField::<G>::zero()).take(length_padding));
 
+            /*
             // zk-rows
             for row in w.iter_mut().rev().take(ZK_ROWS as usize) {
                 *row = <ScalarField<G> as UniformRand>::rand(rng);
-            }
+            } */
```

Comment:
I guess I don't understand why removing these and adding the zk rows manually on the circuit side, is this a question of control as well for the bba thing?